aspnetcore/fundamentals/openapi/openapi-comments.md

```diff
@@ -0,0 +1,307 @@
+* XML comments from the target assembly's own code via a [`ParseCompilation`](https://source.dot.net/#Microsoft.AspNetCore.OpenApi.SourceGenerators/XmlCommentGenerator.Parser.cs,45358a4d0fff76ac) implementation.
+
+The distinction between these two sources is important. XML documentation files passed as `AdditionalFiles` are static. XML comments from the target assembly come from Roslyn's <!-- review `XmlDocumentationCommentProvider`--> [XML documentation comments provider](https://github.com/dotnet/roslyn/blob/main/src/Compilers/Core/Portable/MetadataReference/PortableExecutableReference.cs#L48-L62). The XML documentation comments provider enhances functionality for connecting an XML comment to the compilation symbol's that it's associated with. This has implications for the way `<inheritdoc />` resolution happens in the implementation, discussed in the next section.
+
+XML comments are parsed into structured `XmlComment` objects with:
+
+* Summary, description, remarks, returns, value sections.
+* Parameter documentation with name, description, examples.
+* Response documentation with status codes and descriptions.
+* Support for examples and deprecated markers.
+
+The `XmlComment` class processes [XML documentation tags](https://learn.microsoft.com/dotnet/csharp/language-reference/xmldoc/recommended-tags) like: `<c>`, `<code>`, `<list>`, `<para>`, `<paramref>`, `<typeparamref>`, `<see>`, and `<seealso>`. For XML documentation tags that use references to other elements, like `<see cref="SomeOtherType">`, the implementation strips out the XML tag and maps the reference to plain text for inclusion in the OpenAPI document.
+
+### Support for `<inheritdoc/>`
+
+<!-- original - review my change 
+`<inheritdoc />` tags present an important opportunity because they indicate that comments must be resolved from a base class or implemented interface. 
+The source generator uses its knowledge of the symbol's present in the compilation to discover base classes and interfaces associated with the symbol a given `<inheritdoc />` is placed on and supports resolving them automatically
+-->
+
+`<inheritdoc />` tags indicate that comments must be resolved from a base class or implemented interface. When `<inheritdoc />` is placed on a symbol, The source generator:
+
+* Uses its knowledge of the symbol in the compilation to discover base classes and interfaces associated with the symbol.
+* Supports resolving them automatically
+
+The automatic resolution behavior is currently available for XML documentation comments that exist in the assembly under compilation, and ***not*** XML documentation tags that are in referenced projects or packages. In the later scenario, XML documentation comments:
+
+* Are only presented as text.
+* There's no trivial strategy for:
+  * Associating the text content to compilation symbols.
+  * Developing an understanding of the inheritance hierarchy associated with the types.
+
+### Member Identification
```

Comment:
I think we can remove the content about `MemberKey` here. It was a point in-time implementtion detail and was actually removed for Preview 4 via https://github.com/dotnet/aspnetcore/pull/61145.